boltdesignsystem/bolt
# DS-235 Remove IE11 polyfills from PL

## Jira

https://pegadigitalit.atlassian.net/browse/DS-235

## Summary

Remove IE11 polyfills from the PatternLab build, similar to #1992.

## Details

This PR drops IE support in the Pattern Lab build itself. It has no effect on code downstream, just removes the parts we no longer need.

## How to test

- Check out locally.
- Verify PL builds without issue
- No regressions to PatternLab viewer or custom PL components.
